src/compiler/scala/tools/nsc/ObjectRunner.scala

```diff
@@ -21,9 +21,9 @@ trait CommonRunner {
   /** Run a given object, specified by name, using a
    *  specified classpath and argument list.
    *
-   *  @throws ClassNotFoundException
-   *  @throws NoSuchMethodException
-   *  @throws InvocationTargetException
+   *  @throws java.lang.ClassNotFoundException
+   *  @throws java.lang.NoSuchMethodException
+   *  @throws java.lang.reflect.InvocationTargetException
```

Comment:
This gets an exceptions thrown clause but no exceptions. That is not your fault, I don't know how it works but was curious to pick something and see what it looked like. https://ashawley.github.io/scala/2.13.0/compiler/scala/tools/nsc/ObjectRunner$.html

Reply:
Yeah, you found a good one.  I don't know what that is about...  Experimenting with it further, it would seem it's because there is no text description after the exception class name.  I fixed the warning about "unable to find link" by adding `java.lang`, but it seems we need a new warning in Scaladoc for when the description is missing.

Reply:
Yup, that's the issue

![Screen Shot 2019-08-20 at 9 55 41 PM](https://user-images.githubusercontent.com/358615/63396521-52e82600-c395-11e9-9e83-02be698a3340.png)